### ISSUE TITLE
Example integration

### ISSUE THREAD

**Issue by guzman-raphael** [imjohnbo/ok-to-test#6]
Came across your solution in trying to find a way to share secrets in a safe way to fork contributors (for CI purposes). Would you mind sharing some example how I could incorporate this into an existing GitHub Actions CI workflow? Still relatively new to actions but solutions for secret access are few and far between. This looks promising!

**Comment by imjohnbo** [comment]
Hi @guzman-raphael! Sure – does [this example](https://github.com/imjohnbo/ok-to-test#example) and the [workflows here](https://github.com/imjohnbo/ok-to-test/tree/master/.github/workflows) help? 🙏

**Comment by fidelcoria** [comment]
I think the problem is users might expect this to be a Action itself. It's not obvious from the readme that I have to copy the workflows into my own repo manually.

**Comment by guzman-raphael** [comment]
Thanks @imjohnbo @fidelcoria for the quick response. Coming from TravisCI and still getting the hang of how things operate in GH Actions. Looking forward to getting this working on a side project I have going on. :)

**Comment by guzman-raphael** [comment]
Hmmm, so it looks like I am having some issues trying to adopt this into a test project I've created [here](https://github.com/guzman-raphael/github-actions-ci-poc). I've created a PR from another user and when I trigger the `PROW` command, the action is triggered but I am getting an `Error: HttpError: Not Found` in the `ok-to-test` logs. Any ideas what could be wrong?

**Comment by fidelcoria-aa** [comment]
Assuming secrets were added correctly (APP_ID, PRIVATE_KEY)
Maybe the app want installed correctly. https://github.com/guzman-raphael/github-actions-ci-poc/settings/installations 
If your app doesn't appear on that page it's not installed on the repo. 
Go to 
https://github.com/settings/apps/GH_APP_NAME/installations 
And install on the repo.

**Comment by guzman-raphael** [comment]
@fidelcoria-aa Thanks! We are getting closer! :smiley:. Did not realize that if I created a GitHub App that it still had to be manually installed. After doing this, it is triggering the expected `ok-to-test-command` action. 

However, there is still a problem. It seems that on fork PR's, it is running `integration.yml` on a checkout of what is in the target branch as opposed to what is being supplied by the fork branch. Have a look at this [new PR](https://github.com/guzman-raphael/github-actions-ci-poc/pull/5) I created. Basically when parsing the results of the test, the modified secret display is not taking effect. Any thoughts why that could be?
